src/cobra/data/mini.json

```diff
@@ -26,9 +26,9 @@
         },
         {
             "annotation": {
-                "ncbigi": [
-                    "GI:1208453",
-                    "GI:1652654"
+                "ncbiprotein": [
```

Comment:
What's the difference between the mini.json in `src/cobra/data` and the one in `tests/data`. Couldn't the tests just use the first one?

Reply:
None.
The tests are using the one in tests, because that's how the functions are designed. The ones in src/cobra/data are example files for users installing cobra without development and without tests.
If we want to have tests rely on the files in src/cobra/data we can rewrite tests and update_pickles.py, which I'd be happy to do in a different PR.

Reply:
That sounds good to me :+1: